### ISSUE TITLE
Creates 1.6 Edge for a 2.1 Spark cluster

### ISSUE THREAD

**Issue by timsmith00** [hdinsight/Iaas-Applications#14]
Is there a way to control what version of HDI this edge is actually using?  When I tried to run this script against an HDI 3.6 (Spark 2.1) cluster - this edge was configured with a 1.6 spark client.

**Comment by timsmith00** [comment]
touch

**Comment by sjoss** [comment]
Hi , see my pull requests : https://github.com/hdinsight/Iaas-Applications/pull/15

**Comment by aaneja** [comment]
No, the version of HDI on the edgenode is the same as the rest of the cluster. Can you still repro this problem ?
